### PR TITLE
Fixes: model name 

### PR DESCRIPTION
**Pull request by Dineshjnld**
- removed the duplicated decenter in dowloaded file.
- model name updated based on  archive name if its not already set

### CODE REVIEW

**Comment by Dineshjnld** [comment]
sample

File: app.py [hunk 1]
```diff
@@ -100,7 +102,12 @@ def setDemoMode(val: bool = False):
     type=["zip"],
     on_change=lambda: setDemoMode(False),
 )
-
+#if app.model_name_changed and input_archive : Fixme
```
Comment (Nasfame) [inline app.py:105]: fix this

File: app.py [hunk 2]
```diff
@@ -273,10 +280,10 @@ def setDemoMode(val: bool = False):
             st.download_button(
                 label="Download Working Directory",
                 data=f1,
-                file_name=f"decenter-{os.path.basename(zipfile_)}",
+                file_name=f"{os.path.basename(zipfile_)}",
```
Comment (Nasfame) [inline app.py:283]: gud

**Review by Nasfame — CHANGES_REQUESTED** [review]

**Review by Dineshjnld — COMMENTED** [review]
done

File: app.py [hunk 3]
```diff
@@ -45,19 +45,25 @@ class App:
     version: str = "v3"
     demo: bool = True
     model_name: str = "decenter-model-linear-reg-sample_v3"
-
+    model_name_changed: bool = False
+    
+    def set_model_name(self, model_name:str):
+        app.model_name=model_name
+        app.model_name_changed=True
     def validate_model_name(self):
```
Comment (Nasfame) [inline app.py:53]: add a blank line before defining this method

**Review by Nasfame — CHANGES_REQUESTED** [review]
minor change: req

File: app.py (hunk 3 above)
Comment (Dineshjnld) [reply app.py:54]: done

**Review by Nasfame — APPROVED** [review]
LGTM

**Pull request merged**